rfc8446.xml

```diff
@@ -3511,23 +3912,24 @@ content type of the message.
 This padding scheme was selected because it allows padding of any encrypted
 TLS record by an arbitrary size (from zero up to TLS record size
 limits) without introducing new content types.  The design also
-enforces all-zero padding octets, which allows for quick detection of
+enforces all-zero padding octets; this allows for quick detection of
 padding errors.</t>
 
 <t>Implementations MUST limit their scanning to the cleartext returned
 from the AEAD decryption.  If a receiving implementation does not find
 a non-zero octet in the cleartext, it MUST terminate the
-connection with an “unexpected_message” alert.</t>
+connection with an "unexpected_message" alert.</t>
 
-<t>The presence of padding does not change the overall record size limitations
-- the full encoded TLSInnerPlaintext MUST NOT exceed 2^14 + 1 octets. If the
-maximum fragment length is reduced, as for example by the max_fragment_length
-extension from <xref target="RFC6066"></xref>, then the reduced limit applies to the full plaintext,
-including the content type and padding.</t>
+<t>The presence of padding does not change the overall record size
+limitations: the full encoded TLSInnerPlaintext MUST NOT exceed 2^14 + 1
+octets. If the maximum fragment length is reduced -- as, for example, by the
+max_fragment_length extension from <xref target="RFC6066"></xref> --
```

Comment:
We also see that RFC 6066 is linked to max_fragment_length twice in Section 4.2 and once in Appendix B.3.1.  Should these three entries point to draft-ietf-tls-record-limit as well?
A wrinkle:  RFC 6066 is currently listed as a Normative Reference.  For now, we have added draft-ietf-tls-record-limit as an Informative Reference ("[TLS-Record-Limit]"), per our requirement that draft RFCs cannot be Normative References.
Alternatively, since draft-ietf-tls-record-limit is in our queue but its IANA actions are not yet completed, perhaps Benjamin Kaduk as AD could request expedited processing if IANA is able to complete the actions soon?  We could then publish these two documents together.

Reply:
We've been in extensive dialogue with IANA about the TLS-related registry actions in flight.  This document's actions are done, the separate "IANA registry updates" draft actions are almost done, and the draft-ietf-tls-record-limit actions should be completed shortly thereafter.